content/billing/core-billing/buy-core.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Buy Docker Core
+description: Learn how to start your Docker Core subscription.
+keywords: payments, billing, subscription, invoices, docker core, buy docker
+---
+
+Docker Core subscriptions offer features and benefits to support both new and professional developers, as well as plans for individuals, teams, and enterprise businesses. To learn more about what's included with each plan, see [Docker Core subscriptions and features](../../subscription/core-subscription/details.md) and [Docker Pricing](https://www.docker.com/pricing/) to compare plans.
+
+In this section, learn how to get started with a Docker Core subscription for individuals or for organizations. Before you begin, make sure you have a [Docker ID](../../docker-id/_index.md).
```

Comment:
**question:** same as above. Is it `Docker Core subscription` vs `Docker Core plan`?

Reply:
Good question, I would opt for `subscription` since that includes the product tier and the plan.